server/app.js

```diff
@@ -15,16 +15,63 @@ app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(cors());
 
+const calendarModel = require('./src/models/calendar.js');
+
+app.get('/', async (req, res) => {
+  try {
+    const calendar = await calendarModel.find();
+    res.json(calendar);
+  } catch (err) {
+    res.status(500).json({ message: err.message });
+  }
+});
+
+app.post('/add-event', async (req, res) => {
+  calendarModel.create(req.body).then((calendar) => {
+    res.json(calendar);
+  });
+});
+
+app.delete('/delete-event/:eventId', async (req, res) => {
+  const eventId = req.params.eventId;
+  calendarModel.findByIdAndDelete({ _id: eventId }).then((calendar) => {
+    res.json(calendar);
+  });
+});
+
+app.put('/update-event/:eventId', async (req, res) => {
+  const eventId = req.params.eventId;
+  calendarModel
+    .findByIdAndUpdate({ _id: eventId }, req.body)
+    .then((calendar) => {
+      res.json(calendar);
+    });
+});
+
 // MongoDB connection
 mongoose
-  .connect(process.env.MONGODB_URI)
-  .then(() => console.log("MongoDB connected"))
-  .catch((err) => console.log(err));
+  .connect(
+    'mongodb+srv://admin:1234@cluster0.b8ga8wv.mongodb.net/calendar?retryWrites=true&w=majority',
```

Comment:
use .env file

Reply:
I have used a .env variables where appropriate on the client side but for some reason, the website breaks when I add .env variables on the server side. There is just this one MongoDB connection string I need to put in the .env file.